src/Microsoft.AspNetCore.ResponseCaching/Streams/ResponseCachingStream.cs

```diff
@@ -183,19 +183,12 @@ public override void WriteByte(byte value)
             }
         }
 
-#if NETSTANDARD1_3
-        public IAsyncResult BeginWrite(byte[] buffer, int offset, int count, AsyncCallback callback, object state)
```

Comment:
Yes, we need to override Begin/End stream methods. These were not in early standards.